python/lsst/obs/base/_fitsRawFormatterBase.py

```diff
@@ -21,27 +21,32 @@
 
 __all__ = ("FitsRawFormatterBase",)
 
-from abc import ABCMeta, abstractmethod
+from abc import abstractmethod
 from deprecated.sphinx import deprecated
 
-from astro_metadata_translator import ObservationInfo
+from astro_metadata_translator import fix_header, ObservationInfo
 
 import lsst.afw.fits
 import lsst.afw.geom
 import lsst.afw.image
-from lsst.daf.butler import FileDescriptor
+from lsst.daf.butler import FileDescriptor, Formatter
 import lsst.log
 
-from .formatters.fitsExposure import FitsImageFormatterBase
 from .makeRawVisitInfoViaObsInfo import MakeRawVisitInfoViaObsInfo
 from .utils import createInitialSkyWcsFromBoresight, InitialSkyWcsError
 
 
-class FitsRawFormatterBase(FitsImageFormatterBase, metaclass=ABCMeta):
+class FitsRawFormatterBase(Formatter):
     """Abstract base class for reading and writing raw data to and from
     FITS files.
     """
 
+    supportedExtensions = frozenset({".fits", ".fits.gz", ".fits.fz", ".fz", ".fit"})
+    extension = ".fits"
+
+    unsupportedParameters = {}
+    """Support all parameters."""
```

Comment:
This is a bit of a lie - the raw formatters have never supported subimage loads, and I've made that explicit in a commit on this branch (I think the behavior before was to silently ignore `bbox` and `origin` parameters).

But IIRC, this attributes is actually used by assembly/disassembly to know whether a component _StorageClass_ accepts parameters passed when loading the parent, and hence I shouldn't actually remove `bbox` and `origin` here.  If that's correct, maybe we should remove this attribute from the base `Formatter` and have that logic look at component StorageClass components instead?  Is there some subtle reason why we can't do that either?